packages/components/src/icon/README.md

```diff
@@ -50,9 +58,7 @@ const MyIcon = () => (
 ```jsx
 import { Icon } from '@wordpress/components';
 
-const MyIcon = () => (
-	<Icon icon="screenoptions" className="example-class" />
-);
```

Comment:
I see still string here. Is it correct?

Reply:
yes, this one will be kept for Backward compatibility (this component as well). I think we may remove the component from the handbook entirely at some point.